infra/aws/terraform/cncf-k8s-infra-aws-capa-ami/iam-imagebuilder.tf

```diff
@@ -0,0 +1,97 @@
+}
+
+output "imagebuilder_accesskey" {
+  value = aws_iam_access_key.imagebuilder.encrypted_secret
+}
+
+output "imagebuilder_keyid" {
+  value = aws_iam_access_key.imagebuilder.id
+}
+
+resource "aws_iam_user_policy_attachment" "test-attach" {
+  user       = aws_iam_user.imagebuilder.name
+  policy_arn = aws_iam_policy.imagebuilder.arn
+}
+
+resource "aws_iam_policy" "imagebuilder" {
+  name        = "capa-image-builder-policy"
+  path        = "/"
+  description = "Minimum required policy for image builder"
```

Comment:
The actual policy seems broader than the description suggests.

Reply:
These are the permissions required for Packer when creating AMIs: https://developer.hashicorp.com/packer/integrations/hashicorp/amazon#iam-task-or-instance-role

Reply:
Sure, but I actually don't agree with Hashicorp here.

Check out https://blog.stefan-koch.name/2021/05/16/restricted-packer-aws-permissions

Reply:
I haven't seen that before, makes sense. Thats great, thanks for sharing that @sftim.